proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -267,6 +272,150 @@ message SelectOrUpdateRequest {
   repeated string column_specs = 3;
 }
 
+message Expression {
+  oneof type {
+    string column_name = 1;
+    int64 long_value = 2;
+    string raw = 3;
```

Comment:
why only raw and long_value? if various other numbers/etc can be raw, why can't long?

Also it seems odd to refer to the "type" as being "column_name" or "raw" - perhaps expression could have a "value" which could be "column reference" or "long literal", "raw X value", etc?

Reply:
Just found the Expression type hierarchy in existing code, and I'm only more confused - why is RawString not a Value, but ColumnName is? Shouldn't ColumnName be a "Reference" or something rather than a Value?

Reply:
column name, long value, and raw are the only plumbed through parts at the table api layer right now, but I agree we should flesh these out more (https://github.com/deephaven/deephaven-core/issues/830). I can complete the loop a bit here in these regards.

As it is right now,

Expression = Value | RawString
Value = ColumnName | long (literal)

Maybe we should add more hierarchy here, with "Reference" and "Literal" instead of "Value"

Reply:
I've removed this proto.